src/sentry/utils/audit.py

```diff
@@ -35,3 +51,75 @@ def create_audit_entry(request, transaction_id=None, logger=None, **kwargs):
         logger.info(entry.get_event_display(), extra=extra)
 
     return entry
+
+
+def create_org_delete_log(entry):
+    """ Creates a log of deleted organization """
+    delete_log = DeletedOrganization()
+    delete_log.name = entry.organization.name
+    delete_log.slug = entry.organization.slug
+    delete_log.date_created = entry.organization.date_added
+
+    complete_delete_log(delete_log, entry)
+
+
+def create_project_delete_log(entry):
+    """ Creates a log of deleted project """
+    delete_log = DeletedProject()
+
+    project = Project.objects.get(id=entry.target_object)
```

Comment:
Let's add in a `select_related('team')` since this also is being used below. That prevents an extra query.